ui/.eslintrc

```diff
@@ -33,6 +33,7 @@
     ],
     "import/prefer-default-export": 0,
     "linebreak-style": 0,
-    "jsx-a11y/anchor-is-valid": 0
+    "jsx-a11y/anchor-is-valid": 0,
+    "no-mixed-operators": 0
```

Comment:
This is a rule to help us make good use of operators. pls do not disable it, instead, you should follow the rule.

Reply:
Use eslint-disable-line instead of disable it globally.